satpy/tests/reader_tests/test_seviri_l2_bufr.py

```diff
@@ -38,32 +41,77 @@
 }
 
 DATASET_INFO = {
+    'name': 'testdata',
     'key': '#1#brightnessTemperature',
+    'coordinates': ('longitude', 'latitude'),
     'fill_value': 0
 }
 
+DATASET_INFO_LAT = {
+    'name': 'latitude',
+    'key': 'latitude',
+    'fill_value': -1.e+100
+}
+
+DATASET_INFO_LON = {
+    'name': 'longitude',
+    'key': 'longitude',
+    'fill_value': -1.e+100
+}
+
+
 DATASET_ATTRS = {
     'platform_name': 'MET08',
     'ssp_lon': 41.5,
     'seg_size': 16
 }
 
+AREA_DEF = geometry.AreaDefinition(
+    'msg_seviri_iodc_48km',
+    'MSG SEVIRI Indian Ocean Data Coverage service area definition with 48 km resolution',
+    "",
+    {'a': 6378169., 'b': 6356583.8, 'lon_0': DATASET_ATTRS['ssp_lon'],
+     'h': 35785831., 'proj': 'geos', 'units': 'm'},
+    232,
+    232,
+    (-5567248.0742, -5567248.0742, 5567248.0742, 5567248.0742)
+)
+
+
+AREA_DEF_EXT = geometry.AreaDefinition(
```

Comment:
please check this area definition! grid size is defined as 1238, but description refers to 48km grid. something is not right here.

Reply:
indeed a typo, has been fixed now.